extensions/arc/src/test/ui/components/radioOptionsGroup.test.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as should from 'should';
+import * as sinon from 'sinon';
+import { getErrorMessage } from '../../../common/utils';
+import { RadioOptionsGroup, RadioOptionsInfo } from '../../../ui/components/radioOptionsGroup';
+import { loadingError, modelView, radioButtons } from '../../mocks/fakeContainersAndBuilders';
+
+const radioOptionsInfo = <RadioOptionsInfo>{
+	values: [
+		'value1',
+		'value2'
+	],
+	defaultValue: 'value2'
+};
+
+let radioOptionsGroup: RadioOptionsGroup;
+
+describe('radioOptionsGroup', function (): void {
+	beforeEach(async () => {
+		radioOptionsGroup = new RadioOptionsGroup(modelView, (_disposable) => { });
+		await radioOptionsGroup.load(async () => radioOptionsInfo);
+	});
+
+	afterEach(() => {
+		sinon.restore();
+	});
+
+	it('verify construction and load', async () => {
+		should(radioOptionsGroup).not.be.undefined();
+		should(radioOptionsGroup.value).not.be.undefined();
+		radioOptionsGroup.value!.should.equal('value2');
```

Comment:
Add message explaining that the default value should be set to value2

Reply:
done